docs/contributing.rst

```diff
@@ -195,17 +144,23 @@ in a separate branch).
 Installer
 ---------
 
-Installers classes exist to actually setup the certificate and be able
-to enhance the configuration. (Turn on HSTS, redirect to HTTPS,
-etc). You can indicate your abilities through the
-:meth:`~.IInstaller.supported_enhancements` call. We currently only
+Installers classes exist to actually setup the certificate in a server,
+possibly tweak the security configuration to make it more correct and secure
+(Fix some mixed content problems, turn on HSTS, redirect to HTTPS, etc).
+Installer plugins tell the main client about their abilities to do the latter
```

Comment:
I think the wording is confusing. Is there a difference between "Installer classes" and "Installer plugins"?